save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/OrganizationController.kt

```diff
@@ -23,14 +23,14 @@ internal class OrganizationController(private val organizationService: Organizat
      */
     @GetMapping("/get/organization-name")
     fun getOrganizationByName(@RequestParam name: String): Organization =
-            organizationService.findByName(name)
+            organizationService.findByName(name) ?: throw NoSuchElementException("Organization with name [$name] was not found.")
 
     /**
      * @param owner owner name
      * @return a image
      */
     @GetMapping("/avatar")
-    fun avatar(@RequestParam owner: String): ImageInfo? = organizationService.findByName(owner).avatar.let { ImageInfo(it) }
+    fun avatar(@RequestParam owner: String): ImageInfo? = organizationService.findByName(owner)?.avatar.let { ImageInfo(it) }
```

Comment:
Should it be `?.let`? Otherwise function never returns null, only null wrapped into `ImageInfo`. If that's intended, we can change returned type

Reply:
Ok, changed return type.